packages/config-plugins/src/plugins/expo-plugins.ts

```diff
@@ -76,12 +76,14 @@ export const withExpoAndroidPlugins: ConfigPlugin<{
     // AndroidManifest.xml
     AndroidConfig.Package.withPackageManifest,
     AndroidConfig.AllowBackup.withAllowBackup,
+    // Note: The withAndroidIntentFilters plugin must appear before the withScheme
+    // plugin or withScheme will override the output of withAndroidIntentFilters.
+    AndroidConfig.IntentFilters.withAndroidIntentFilters,
```

Comment:
Thanks for this, I'll see about making withScheme not remove schemes since that's undesired behavior now.